src/mixins/filing-template-mixin.ts

```diff
@@ -114,12 +114,7 @@ export default class FilingTemplateMixin extends DateMixin {
         date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
         folioNumber: this.getFolioNumber // folio number, unless overridden below
       },
-      business: {
-        identifier: this.getBusinessId,
-        legalName: this.getNameRequestLegalName,
-        legalType: this.getEntityType,
-        nrNumber: this.getNameRequestNumber
-      },
+      business: this.getEntitySnapshot.businessInfo,
```

Comment:
BUGFIX: some flags were not being saved in the draft, which means they weren't restored later.

A specific example of this is that the `hasRestrictions` flag was True initially, but was empty (undefined) after restoring a draft.